pkg/ddl/split_region.go

```diff
@@ -38,6 +38,8 @@ func splitPartitionTableRegion(ctx sessionctx.Context, store kv.SplittableStore,
 	defer cancel()
 	ctxWithTimeout = kv.WithInternalSourceType(ctxWithTimeout, kv.InternalTxnDDL)
 	if shardingBits(tbInfo) > 0 && tbInfo.PreSplitRegions > 0 {
+		scatter, tableID := getScatterConfig(scatterScope, tbInfo.ID)
+		regionIDs = append(regionIDs, splitIndexRegion(store, tbInfo, scatter, tableID)...)
```

Comment:
Is it for the global index？ Could we add a comment for it？

Besides, do we need to update the capacity for `regionIDs`?

Reply:
Done